sklearn/ensemble/_forest.py

```diff
@@ -960,10 +960,17 @@ class RandomForestClassifier(ForestClassifier):
         <n_jobs>` for more details.
 
     random_state : int, RandomState instance or None, optional (default=None)
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+        Controls both the randomness of the bootstrapping of the samples used
+        when building trees (if ``bootstrap=True``) and the sampling of the
+        features to consider when looking for the best split at each node
+        (if ``max_features < n_features``).
+        Also note that the features are always randomly permuted at each split.
+        Therefore, the best found split may vary, even with the same training
+        data, ``max_features=n_features`` and ``bootstrap=False``, if the
+        improvement of the criterion is identical for several splits enumerated
+        during the search of the best split. To obtain a deterministic
+        behaviour during fitting, ``random_state`` has to be fixed.
```

Comment:
I think we can omit this last part (from "Also note that..." to "has to be fixed").

The rest above is perfect

Reply:
This note was already there. I just moved it. Should I still remove it?

Reply:
I see, maybe just leave it where it was in the notes then